src/proxy-worker.ts

```diff
@@ -23,12 +23,13 @@ import * as nodeTunnel from 'node-tunnel';
 import * as winston from 'winston';
 
 import { captureException, device, errors, getLogger, Metrics } from './utils';
+import { intVar } from './utils/config';
 
 const VPN_SERVICE_API_KEY = Buffer.from(process.env.VPN_SERVICE_API_KEY!);
-const VPN_CONNECT_PROXY_PORT = parseInt(
-	process.env.VPN_CONNECT_PROXY_PORT!,
-	10,
-);
+// listen port
+const VPN_CONNECT_PROXY_PORT = intVar('VPN_CONNECT_PROXY_PORT', 3128);
```

Comment:
I think you can just follow the current convention of using `parseInt` here (or change all other the port params to use `intVar` too, if you'd rather… in which case I'd argue a `portVar` function would be worthwhile with additional validity checks).

Reply:
OK, I'll see what I can do.